pkg/cloudprovider/gce/gce.go

```diff
@@ -241,7 +241,9 @@ func (gce *GCECloud) CreateTCPLoadBalancer(name, region string, externalIP net.I
 	if err != nil {
 		return nil, err
 	}
-	return net.ParseIP(fwd.IPAddress), nil
+	loadBalancerInfo := &api.LoadBalancerInfo{Rewrite: false}
```

Comment:
Why doesn't this value of rewrite come from the service? This suggests a problem with the API to me.